### DISCUSSION TITLE
[Question] About cancelled lecture

### DISCUSSION THREAD

**Discussion by bscho99** [prosyslab-classroom/cs424-program-reasoning#279] [Q&A]
Name: Cho Beomsuk

Hello,
I have a question about the cancelled lecture next week.
I remember from the last lecture that one session is going to be cancelled due to professor's military training,
but couldn't remember exactly when (if it is monday or wednesday).
Is there anyone who remembers when?
Thank you.

**Comment by seo-rii** [comment]
I believe that was monday.

**Comment by KihongHeo** [comment]
Yes. No class on Mon. See you on Wed.

**Comment by bscho99** [comment]
> Yes. No class on Mon. See you on Wed.

Thank you professor! Have a great weekend!
